src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -416,17 +439,18 @@ private void processSampleRecordsForLocation(final long location, final Iterable
 
         // same qualapprox check as Gnarly
         final boolean isIndel = !hasSnpAllele;
-        // System.out.println("KCIBUL -- in the qualapprox w/ " + totalAsQualApprox + " for isIndel " + isIndel + " and SNP:" + SNP_QUAL_THRESHOLD + " and INDEL:" + INDEL_QUAL_THRESHOLD);
         if((isIndel && totalAsQualApprox < INDEL_QUAL_THRESHOLD) || (!isIndel && totalAsQualApprox < SNP_QUAL_THRESHOLD)) {
-            // logger.info(contig + ":" + currentPosition + ": dropped for low QualApprox of  " + totalAsQualApprox);
+            if ( printDebugInformation ) {
```

Comment:
comment about totalAsQualApprox. it is the sum of the QualApprox values for every sample with a variant at this site (but not allele specific). this seems strange. it seems like it should be an average or some other ratio.

Reply:
It's just the straight sum.  I agree it seems like you would want a single high quality variant or some such... but this is exactly what is calculated in the warp pipeline.